test/UniswapV3Staker.unit.spec.ts

```diff
@@ -602,36 +613,41 @@ describe('UniswapV3Staker.unit', async () => {
         claimDeadline,
       })
 
-      subject = ({ to }) =>
-        staker.connect(wallets[0]).unstakeToken({
+      subject = () =>
+        staker.connect(lpUser0).unstakeToken({
           creator: incentiveCreator.address,
           rewardToken: rewardToken.address,
           tokenId,
           startTime,
           endTime,
           claimDeadline,
-          to,
         })
     })
 
-    const recipient = wallets[3].address
-
     describe('works and', async () => {
       it('decrements numberOfStakes by 1', async () => {
-        await subject({ to: recipient })
+        await subject()
```

Comment:
not originally from you, but I think we should either complete this test or add a TODO (since it's not pending)